numpy/_core/overrides.py

```diff
@@ -21,6 +22,11 @@
 
 def get_array_function_like_doc(public_api, docstring_template=None):
```

Comment:
I realized that changing the default value for docstring_template to an empty string would probably work too, but I cannot test it right now (afk).